### PR TITLE
KDP-2724 

### PR DESCRIPTION
**Pull request by bmcrea**
Updated Spark connector example to 4.0.3 and added additional example instructions.

### CODE REVIEW

File: spark-connector/README.md [hunk 1]
```diff
@@ -22,8 +22,10 @@ $SPARK_HOME/bin/spark-submit ./target/kdp-spark-example.jar --master local[4]
 
 ### KDP Count
 
-This example, in Java, covers the main functionality of the KDP Spark Connector. It includes read, write, and simple ABAC. You will need Spark 3, Maven 3, and Java 8 or Java 11 to run this example.
+This example, in Java, covers the main functionality of the KDP Spark Connector. It includes read, write, and simple ABAC. You will need Spark 3, Maven 3, and Java 8 11 to build and run this example.
```
Comment (tzervas) [inline spark-connector/README.md]: Is this indicating Java 8.11, or would it be suggesting Java 8 or 11?

**Review by tzervas — COMMENTED** [review]
Requesting clarification on line 25 of spark-connector/README.md

**Review by cch-k — APPROVED** [review]
Approving ahead. After Tyler's comment is addressed.

**Review by danielhumfleet — APPROVED** [review]

File: spark-connector/README.md (hunk 1 above)
Comment (bmcrea) [reply spark-connector/README.md]: It should be Java 8 or 11. I've fixed it - thanks for catching that.

**Review by tzervas — APPROVED** [review]

**Pull request merged**